### PR TITLE
kubelet summary stats api reports if imagefs is same as rootfs

### PR DESCRIPTION
**Pull request by derekwaynecarr**
Adds the ability to know if the runtime imagefs is the same as the node rootfs.

### CODE REVIEW

**Comment by derekwaynecarr** [comment]
This is needed to know how to rank pods for eviction when low on imagefs in disk based eviction.

I am not sure I love the field name, so I am open to other naming input.

/cc @kubernetes/sig-node @kubernetes/rh-cluster-infra

File: pkg/kubelet/api/v1alpha1/stats/types.go [hunk 1]
```diff
@@ -56,6 +56,8 @@ type RuntimeStats struct {
 	// This filesystem could be the same as the primary (root) filesystem.
 	// Usage here refers to the total number of bytes occupied by images on the filesystem.
 	ImageFs *FsStats `json:"imageFs,omitempty"`
+	// This value is true if the filesystem where container images are stored is the same as the primary (root) filesystem.
```
Comment (vishh) [inline pkg/kubelet/api/v1alpha1/stats/types.go]: Do we have to expose this value via the REST API?

Comment (derekwaynecarr) [inline pkg/kubelet/api/v1alpha1/stats/types.go]: In theory, we could keep it internal, but what's the concern?

Comment (vishh) [inline pkg/kubelet/api/v1alpha1/stats/types.go]: Concern is the scope of the summary API. A lot of effort went in to ensure that its surface area was minimal.

**Comment by derekwaynecarr** [comment]
Closing this in favor of exposing it on the internal interface instead.

**Pull request closed without merging**